shared/devices/device-page/index.js

```diff
@@ -0,0 +1,77 @@
+import React, {Component} from 'react'
+import {bindActionCreators} from 'redux'
+import {connect} from 'react-redux'
+import moment from 'moment'
+
+import Render from './index.render'
+import RemoveDevice from '../device-revoke'
+
+import * as devicesActions from '../../actions/devices'
+import {routeAppend} from '../../actions/router'
+
+export default class DevicePage extends Component {
+  static parseRoute (currentPath) {
+    return {
+      componentAtTop: {
+        title: 'Device page',
+        props: {
+          device: currentPath.get('device')
+        }
+      },
+      subRoutes: {
+        removeDevice: RemoveDevice
+      }
+    }
+  }
+
+  _buildTimeline (device) {
```

Comment:
The core doesn't yet send us other info we could show in the timeline, such as Last Seen.